src/test/java/trailerRegistration/PostTrailerRegistration.java

```diff
@@ -265,4 +276,275 @@ public void testTrailerUnRegistrationForIntReasonForDeRegistration() {
         trailerRegistrationSteps.putTrailerRegistrationWithAlterations(randomTrn,trailerDeRegistrationRecord,DeRegisterAlterations);
         trailerRegistrationSteps.statusCodeShouldBe(400);
     }
+
+    @Title("CVSB-18921 - AC1 -  VTG5A TRN PRESENT - TRL Annual Test Pass - With TRN")
+    @Test
+    public void testPostTestResultPassTrailerRegistrationCertificateWithTrn() {
+
+        // Read the base test result JSON
+        String postTechnicalRecord = GenericData.readJsonValueFromFile("technical-records_trl_all_fields.json","$");
+
+        //String randomVin = GenericData.generateRandomVin();
```

Comment:
can you remove the unwanted codes please (the commented codes are there in all tests)